modules/claims_api/lib/claims_api/poa_pdf_constructor/organization.rb

```diff
@@ -28,11 +28,8 @@ def page1_signatures(_signatures)
         []
       end
 
-      def page2_signatures(signatures)
-        [
-          ClaimsApi::PoaPdfConstructor::Signature.new(data: signatures['veteran'], x: 35, y: 263),
-          ClaimsApi::PoaPdfConstructor::Signature.new(data: signatures['representative'], x: 35, y: 216)
-        ]
+      def page2_signatures(_signatures)
```

Comment:
I think this change would impact new V1 submissions

Reply:
good catch.